.github/workflows/End2EndTestApacheAws.yml

```diff
@@ -60,8 +60,10 @@ jobs:
         ./build_apache.sh ../../snowflake-kafka-connector package
 
     - name: End to End Test of Apache Plarform 2.5
+      uses: nick-invision/retry@v2
```

Comment:
Is this action github verified on the github action market place? We are only allowed to use those verified actions.

Reply:
Yes this was from marketplace..

Reply:
I searched for it on github action marketplace, the developer is not verified by github. So we cannot use this one.
![Screen Shot 2021-06-09 at 12 20 36 AM](https://user-images.githubusercontent.com/63482288/121310830-a1dbb400-c8b8-11eb-88e9-0ea11d7bbf6f.png)
![Screen Shot 2021-06-09 at 12 20 50 AM](https://user-images.githubusercontent.com/63482288/121310840-a4d6a480-c8b8-11eb-85bc-91df5625d4b5.png)

Reply:
Ahh I see.. No worries. I will meddle with other settings like max.poll.time.ms because I saw those errors popping up multiple times.